src/config/config.ts

```diff
@@ -663,6 +663,10 @@ export class Config implements IConfig {
     const SHELL = process.env.SHELL ?? osUserInfo().shell?.split(sep)?.pop()
     if (SHELL) {
       shellPath = SHELL.split('/')
+    } else if (this.windows && process.title.toLowerCase().includes('powershell')) {
+      shellPath = ['pwsh']
```

Comment:
the `process.title` check works on windows but powershell can be used on mac/linux where the title is the node bin path.

<img width="897" alt="Screenshot 2024-07-08 at 3 02 59 PM" src="https://github.com/oclif/core/assets/6853656/4a3e466e-b98a-40bd-912c-5a1fb2a3da8c">

what if we look for a powershell-specific env var to detect it on any OS?
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4

there are a few windows-specific but `PSModulePath` is set on all OSes by default:
> The $env:PSModulePath environment variable contains a list of folder locations that are searched to find modules and resources. On Windows, the list of folder locations is separated by the semi-colon (;) character. On non-Windows platforms, the colon (:) separates the folder locations in the environment variable.

mac:
<img width="1782" alt="Screenshot 2024-07-08 at 3 07 20 PM" src="https://github.com/oclif/core/assets/6853656/271595af-14c4-4806-867e-504b5232dc50">

win:
![Screenshot 2024-07-08 at 3 09 01 PM](https://github.com/oclif/core/assets/6853656/337effd4-243f-46c6-a06b-1457f98dd9bc)

Reply:
follow up:
`PSModule` is also available in `cmd` ... will be using both checks.